js/exmo.js

```diff
@@ -492,6 +494,94 @@ module.exports = class exmo extends Exchange {
         return result;
     }
 
+    async fetchDepositWithdrawFees (codes = undefined, params = {}) {
```

Comment:
@samgermain I think we should fill in these values with the default network fees (btc in this case)
![image](https://user-images.githubusercontent.com/43336371/203981003-634b71ba-5f15-4992-bf69-8b731d4ee671.png)